quickstart/etcdmigration.md

```diff
@@ -0,0 +1,267 @@
+---
+Title: ETCD Migration Procedure
+---
+
+By following the given steps, you can successfully migrate ETCD from one node to another during maintenance activities like node drain etc., ensuring the continuity and integrity of the ETCD data.
+
+:::note
+Take a snapshot of the ETCD. Click [here](https://etcd.io/docs/v3.3/op-guide/recovery/) for the detailed documentation.
+:::
+
+
+## Step 1: Draining the ETCD Node
+
+1. Assuming we have a three-node cluster with three ETCD replicas and three pools created in the cluster, verify the ETCD pods and pools with the following commands:
+
+**Command to verify pods**:
+
+{% tabs %}
+{% tab title="Command" %}
+
+```text
+kubectl get pods -n mayastor -l app=etcd -o wide
+```
+{% endtab %}
+{% tab title="Output" %}
+
+```text
+NAME              READY   STATUS    RESTARTS   AGE     IP             NODE       NOMINATED NODE   READINESS GATES
+mayastor-etcd-0   1/1     Running   0          4m9s    10.244.1.212   worker-1   <none>           <none>
+mayastor-etcd-1   1/1     Running   0          5m16s   10.244.2.219   worker-2   <none>           <none>
+mayastor-etcd-2   1/1     Running   0          6m28s   10.244.3.203   worker-0   <none>           <none>
+```
+{% endtab %}
+{% endtabs %}
+
+**Command to verify pools**:
+
+{% tabs %}
+{% tab title="Command" %}
+
+```text
+kubectl get dsp -n mayastor
+```
+{% endtab %}
+{% tab title="Output" %}
+
+```text
+NAME     NODE       STATE    POOL_STATUS   CAPACITY       USED          AVAILABLE
+pool-0   worker-0   Online   Online        374710730752   22561161216   352149569536
+pool-1   worker-1   Online   Online        374710730752   21487419392   353223311360
+pool-2   worker-2   Online   Online        374710730752   21793603584   352917127168
+```
+{% endtab %}
+{% endtabs %}
+
+2. From etcd-0/1/2 we could see all the values are registered in database, once we migrated etcd to new node, all the key-value pairs should be available across all the pods. Run the following commands from any etcd pod.
+
+**Command to get etcd data**:
+
+{% tabs %}
+{% tab title="Command" %}
+
+```text
+root@543-master:~# kubectl exec -it mayastor-etcd-0 -n mayastor -- bash
+Defaulted container "etcd" out of: etcd, volume-permissions (init)
+I have no name!@mayastor-etcd-0:/opt/bitnami/etcd$ ETCDCTL_API=3
+I have no name!@mayastor-etcd-0:/opt/bitnami/etcd$ etcdctl get --prefix ""
+```
+{% endtab %}
+{% tab title="Output" %}
+
+```text
```

Comment:
Why do we need to print all the data here @datacore-tilangovan ?

Reply:
Its a output section of the command just to verify we have all the data similar to other members. May be we can just put the command alone as output varies depends on the end user environment.